deploy/olm-catalog/0.3.0/submarineraddon.open-cluster-management.io_submarinerconfigs.yaml

```diff
@@ -35,6 +35,9 @@ spec:
             IPSecNATTPort:
               description: IPSecNATTPort represents IPsec NAT-T port (default 4500).
               type: integer
+            NATTDiscoveryPort:
+              description: NATTDiscoveryPort represents NAT-T Discovery port (default UDP/4900).
+              type: integer
```

Comment:
I don’t think the 0.3.0 catalog entry should be changed.

Reply:
Yeah, I wasn't sure of this either, but this change was already done in fix for VxLan and NattEnable. Should we remove those from 0.3.0 too?

Reply:
Ah yes, I’ve pushed https://github.com/open-cluster-management/submariner-addon/pull/126 to address that.